automated/linux/deqp-runner/deqp-runner.yaml

```diff
@@ -0,0 +1,48 @@
+metadata:
+    format: "Lava-Test Test Definition 1.0"
+    name: deqp-runner
+    description: "The drawElements Quality Program (deqp) Testing allows to test OpenGL API's, uses
+                  OpenGL ES CTS from https://github.com/KhronosGroup/VK-GL-CTS."
+
+    maintainer:
+        - anibal.limon@linaro.org
+    os:
+        - openembedded
+    scope:
+        - functional
+    devices:
+        - dragonboard410c
+        - dragonboard820c
+        - dragonboard845c
+
+params:
+    DISPLAY: ":0"
+    EGL_PLATFORM: "surfaceless"
+    DEQP_BIN: "deqp-gles2"
+    DEQP_CASES: ""
+    DEQP_FAIL: ""
+    DEQP_EXCLUDE: ""
+    DEQP_RUNNER_OPTIONS: "--compact-display false --shuffle false --allow-flakes true"
+    DEQP_RUNNER_JOBS: ""
+    DEQP_OPTIONS: "--deqp-surface-width=256 --deqp-surface-height=256 --deqp-surface-type=pbuffer --deqp-gl-config-name=rgba8888d24s8ms0 --deqp-visibility=hidden --deqp-shadercache=disable"
+
+run:
+    steps:
+        - export DISPLAY=${DISPLAY}
+        - export EGL_PLATFORM=${EGL_PLATFORM}
+        - export RESULT_FILE=result.txt
+        - cd ./automated/linux/deqp-runner
+        - set +e
+        - deqp-runner --deqp ${DEQP_BIN} --output $RESULT_FILE ${DEQP_CASES} ${DEQP_FAIL} ${DEQP_EXCLUDE} ${DEQP_RUNNER_OPTIONS} ${DEQP_RUNNER_JOBS} -- ${DEQP_OPTIONS}
+        - DEQP_EXITCODE=$?
+        - set -e
+        - test_set_name="$(basename ${DEQP_BIN})"
+        - lava-test-set start "$test_set_name"
```

Comment:
please use sh-test-lib functions instead of calling lava-* helpers directly. This test should also run nicely standalone and with test-runner.py.

Reply:
@mwasilew There is no lava-test-set function inside sh-test-lib.

Reply:
so add it :)